src/js/common/model/veteran.js

```diff
@@ -1,7 +1,8 @@
 'use strict';  // eslint-disable-line
 // Veteran resource prototype objects. In common so server unittests can access.
 
-const _ = require('lodash');
+const _includes = require('lodash/includes');
```

Comment:
@awong-dev I forgot I left this in, actually. Some people prefer using _ here to note that the function comes from Lodash, instead of having a bunch of unnamespaced functions in their code. I think I prefer just the plain function name, though, so I'll remove it.